dub.sdl

```diff
@@ -211,6 +211,20 @@ configuration "plugin_uml_debug" {
     }
 }
 
+configuration "plugin_fuzz" {
+    dependency "dextool:dsrcgen" version="*"
+    dependency "dextool:libclang" version="*"
+    dependency "dextool:clang" version="*"
+    dependency "dextool:cpptooling" version="*"
+
+    targetName "dextool-fuzz"
+    targetType "executable"
+    
+    sourcePaths "plugin/source" "plugin/fuzz"
+    importPaths "source" "plugin/source" "plugin/fuzz/source"
+    excludedSourceFiles "source/application/*" "source/test/*" "source/devtool/*" "source/cpptooling/*"
+}
+
```

Comment:
I recommend to also add a "plugin_fuzz_debug" with code contracts etc activated. It makes it a lot easier to debug but at the same time keep the release binary "fast, efficient, not-so-chatty".